### DISCUSSION TITLE
Where to get help?

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#2841] [General Questions & Discussion] [answered]
Where to get help from? in Discord it tells you to look in the Docs. Posting in the Discord -  a bot tells you to got to here. Is that right?

**Comment by robinkwilson** [comment, marked as answer]
Here is correct! Post your questions here so the question/answer is searchable to other users.

**Reply by brunomartelli** [reply]
Cool thnx

**Comment by brianpeiris** [comment]
Sorry for the confusion. We're transitioning away from Discord-based help to Github discussions, so there's going to be an awkward period until we decide to move most discussions here.

**Reply by brunomartelli** [reply]
Okidoki! thanks
